samples/GraphicsTester.Portable/Scenarios/RadialGradientInCircle.cs

```diff
@@ -12,18 +12,25 @@ public override void Draw(ICanvas canvas)
 		{
 			canvas.SaveState();
 
-			var paint = new Paint
+			var radialGradientPaint = new RadialGradientPaint
 			{
-				PaintType = PaintType.RadialGradient,
 				StartColor = Colors.White,
 				EndColor = Colors.Black
 			};
 
-			canvas.SetFillPaint(paint, 200, 200, 300, 200);
-			canvas.FillEllipse(100, 100, 200, 200);
+			radialGradientPaint.Center = new Point(0.5, 0.5);
+			radialGradientPaint.Radius = 0.5;
 
-			canvas.SetFillPaint(paint, 250, 500, 100, 500);
-			canvas.FillEllipse(100, 400, 200, 200);
+			var ellipseRect1 = new RectangleF(100, 100, 200, 200);
+			canvas.SetFillPaint(radialGradientPaint, ellipseRect1);
+			canvas.FillEllipse(ellipseRect1);
+
+			radialGradientPaint.Center = new Point(0.6, 0.7);
```

Comment:
Note that the values are relative. It was probably too many changes for this PR, but I think it would be nice to add a property **PaintMappingMode** like https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.brushmappingmode?view=net-5.0

Reply:
I like that idea.